### PR TITLE
firebase deployz and scale everything down by 2

### PR DESCRIPTION
**Pull request by craig-day**
Fix flex issues. we CANNOT use `flex-flow`. Just use `flex-direction` and `flex-wrap`. 
Add some files necessary to deploy to firebase.
Scale everything down by two.

/cc @zendesk/volunteer

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "linksf-dev": "vivid-inferno-4672"
```
Comment (thebritican) [inline .firebaserc]: is this intended?

File: database.rules.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "rules": {
```
Comment (thebritican) [inline database.rules.json]: wat r these for?

File: components/Location/Location.css [hunk 3]
```diff
@@ -4,60 +4,60 @@
...
-  padding: 8px 8px 8px 12px;
-  font-size: 14px;
+  padding: 4px 4px 4px 6px;
+  font-size: 7px;
 }
 
 .insetCall {
-  margin: 20px 0 0 0;
-  border-radius: 6px 6px 0 0;
+  margin: 10px 0 0 0;
```
Comment (thebritican) [inline components/Location/Location.css]: we should probably remove margins and use spacers/padding in a followup PR

File: database.rules.json (hunk 2 above)
Comment (jsdnxx) [inline database.rules.json]: water is for horses

File: .firebaserc (hunk 1 above)
Comment (craig-day) [inline .firebaserc]: eh not really, but I don't think it hurts to have it. I will commit it out and put in a `.example` file I guess.

File: database.rules.json (hunk 2 above)
Comment (craig-day) [inline database.rules.json]: needed to deploy via firebase.

**Comment by roseleaf** [comment]
Font is way too small on list page
Feedback form is also messed up from this

a couple titles are too small on detail page

**Pull request merged**